quarkus/service/src/main/resources/application-it.properties

```diff
@@ -0,0 +1,47 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Configuration specific to integration tests
+# Note: this file MUST be placed in the src/main/resources directory, not src/intTest/resources!
+
+quarkus.http.limits.max-body-size=1000000
+quarkus.http.port=0
+
+quarkus.log.file.path=./build/logs/polaris.log
+
+quarkus.management.port=0
+
+polaris.persistence.type=in-memory
+
+polaris.features.defaults."ALLOW_EXTERNAL_CATALOG_CREDENTIAL_VENDING"=false
```

Comment:
Oh, I see, you set them here. Can we set these from within tests instead?

Reply:
i.e. if a test relies on a certain feature flag being set let's just set it in that test rather than subtlety rely on the default.

Reply:
This file is _only_ for `@QuarkusIntegrationTest` annotated tests (these are in the `intTest` folder). These are "real" black box tests that run the actual final jar in a separate JVM process. Because of that, some tweaks are not possible for these tests, but at least, it is possible to provide a file like this that will be passed to the JVM process.

For `@QuarkusTest` annotated test (in the `test` folder), it is possible to define options that are global to all tests, and also possible to define options that are valid for just some tests, using `QuarkusTestProfile`.

Reply:
> i.e. if a test relies on a certain feature flag being set let's just set it in that test rather than subtlety rely on the default.

That's precisely the intent of this PR :-)

Reply:
Sorry if I misunderstood, I was specifically looking at the comments like:
```
This test expects the server...
```

And thinking that the test shouldn't need that comment and should just always set those exact values

Reply:
I think this comment belongs to another conversation? I'm confused now, what is it that you want me to remove? The javadocs, the README, everything?